next.config.js

```diff
@@ -38,6 +38,16 @@ module.exports = withPlugins([
               'https://media.livepeer.org/play?url=https%3A%2F%2Fcdn.livepeer.com%2Fhls%2F83f6hvl1i9noskxb%2Findex.m3u8',
             permanent: false,
           },
+          {
+            source: '/jobs',
+            destination: 'https://livepeer.com/jobs',
+            permanent: false,
+          },
+          {
+            source: '/jobs/:path*',
```

Comment:
This will 404 because .com uses the teamtailor id for the slug so we'll have to redirect this to https://livepeer.com/jobs as well

Reply:
@FullStackTiger for now let's actually manually create redirects for each individual job 

ie. livepeer.org/jobs/technical-writer -> livepeer.com/jobs/1496366